kornia/utils/draw.py

```diff
@@ -8,6 +8,33 @@
 # TODO: implement width of the line
 
 
+def draw_point2d(image: torch.Tensor, points: torch.Tensor, color: torch.Tensor) -> torch.Tensor:
```

Comment:
```suggestion
def draw_point2d(image: Tensor, points: Tensor, color: Tensor) -> Tensor:
```

from kornia.core import Tensor